R/add_phylopic_tree.R

```diff
@@ -50,11 +59,13 @@ add_phylopic_tree <- function(tree, tip = names(img) %||% names(uuid) %||%
                                 names(name) %||% name,
                               img = NULL,
                               name = if (is.null(img) && is.null(uuid)) tip 
-                                else NULL, 
-                              uuid = NULL, width, padding = NULL,
+                                else NULL,
+                              uuid = NULL, align = "tip",
+                              width, padding = NULL,
                               relWidth = 0.06, relPadding = 1/200,
```

Comment:
The problem with these defaults is that with `align = "plot"`, the silhouettes plot outwith the plotting area and are thus invisible!

If `align = "plot"`, it would be suitable to initialize `relPadding` to `-relWidth`.

We could go
```suggestion
                              relWidth = 0.06, relPadding = if(align == "tip") 1/200 else -relWidth,
```

Or we could set `relPadding = NULL` and include an `if.null(relPadding)` check in the function body, but I like surfacing the default value to the user, particularly in a case like this where the user is likely want to tweak the defaults (and thus to know where they started).

Reply:
Oh geez, thanks for catching this @ms609! I've adapted your suggestion just a little bit by switching the `hjust` when `align = "plot"` and then only switching the sign of the `relPadding` value when `align = "plot"`. It effectively does the same thing as your suggestion, I believe, but it seems more user-friendly to me.